contracts/DXswapDeployer.sol

```diff
@@ -17,19 +20,24 @@ contract DXswapDeployer {
     }
     
     TokenPair[] public initialTokenPairs;
-    
+
+    event FeeReceiverDeployed(address feeReceiver);    
     event FeeSetterDeployed(address feeSetter);
     event PairFactoryDeployed(address factory);
     event PairDeployed(address pair);
         
     // Step 1: Create the deployer contract with all the needed information for deployment.
     constructor(
+        address payable _protocolFeeReceiver,
         address payable _dxdaoAvatar,
+        address _WETH,
         address[] memory tokensA,
         address[] memory tokensB,
         uint32[] memory swapFees
     ) public {
         dxdaoAvatar = _dxdaoAvatar;
+        WETH = _WETH;
+        protocolFeeReceiver = _protocolFeeReceiver;
```

Comment:
protocolFeeReceiver would be the one set to the DXtrust address, right?

Reply:
yes